extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1540,10 +1540,12 @@ export class AmpA4A extends AMP.BaseElement {
   /**
    * Shared functionality for cross-domain iframe-based rendering methods.
    * @param {!JsonObject<string, string>} attributes The attributes of the iframe.
+   * @param {boolean} letCreativeTriggerRenderStart Whether amp-ad-xorigin should let
+   *    the creative trigger render-start, or simply trigger it itself.
    * @return {!Promise} awaiting load event for ad frame
    * @private
    */
-  iframeRenderHelper_(attributes) {
+  iframeRenderHelper_(attributes, letCreativeTriggerRenderStart) {
```

Comment:
Do we need to pass in `letCreativeTriggerRenderStart`? I think we can call `this.letCreativeTriggerRenderStart()` instead.